client/src/app/app.component.html

```diff
@@ -1,2 +1,25 @@
-
-  <router-outlet></router-outlet>
+<nav class="nav-mobile" role="navigation">
+  <div id="menuToggle">
+    <input type="checkbox" />
+    <span></span>
+    <span></span>
+    <span></span>
+    <ul id="menu">
+      <li><a class="menu-item" *ngIf="isLogged()" [routerLink]="['/home']" routerLinkActive="active" >Acceuil</a></li>
+         <li><a class="menu-item" *ngIf="isLogged()" [routerLink]="['/ask']" routerLinkActive="active" >DemandeFeedBack</a></li>
+         <li><a class="menu-item" *ngIf="isLogged()" [routerLink]="['/send']" routerLinkActive="active" >EnvoiFeedBack</a></li>
+         <li><a class="menu-item" (click)="signOut()" class="menu-item">Se deconnecter</a></li>
+       </ul>  
+  </div>
+    <ul><a class="nav-title" routerLink="/home">FeedZback</a></ul>
```

Comment:
```suggestion
    <ul id="menu">
      <li><a class="menu-item" *ngIf="isLogged()" [routerLink]="['/home']" routerLinkActive="active" >Acceuil</a></li>
      <li><a class="menu-item" *ngIf="isLogged()" [routerLink]="['/ask']" routerLinkActive="active" >DemandeFeedBack</a></li>
      <li><a class="menu-item" *ngIf="isLogged()" [routerLink]="['/send']" routerLinkActive="active" >EnvoiFeedBack</a></li>
      <li><a class="menu-item" (click)="signOut()" class="menu-item">Se deconnecter</a></li>
    </ul>  
  </div>
  <ul><a class="nav-title" routerLink="/home">FeedZback</a></ul>
```
Toujours un problème de formatique qui empeche la compréhension du code.